code/extension/ShareThisSTE.php

```diff
@@ -91,50 +124,67 @@ public function ShowShareIcons()
         }
     }
 
-    public function ShareIcons()
+    /**
+     * Get the sharing icons
+     */
+    public function getShareIcons()
     {
         $bookmarks = $this->makeBookmarks('IncludeThisIcon');
         return $this->makeShareIcons($bookmarks);
     }
 
+    /**
+     * Grabbing front end dependencies for the expanded sharing list with some extra
+     * functionality
+     */
     public function ShareAllExpandedList()
     {
-        Requirements::javascript(FRAMEWORK_DIR . '/thirdparty/jquery/jquery.js');
+        Requirements::javascript(SS_SHARETHIS_DIR . '/node_modules/jquery/dist/jquery.min.js');
         Requirements::javascript(SS_SHARETHIS_DIR . '/javascript/ShareAllExpandedList.js');
         $bookmarks = $this->makeBookmarks('IncludeThisIconInExtendedList');
         return $this->makeShareIcons($bookmarks);
     }
 
+    /**
+     * Include share all
+     */
     public function IncludeShareAll()
     {
         $config = $this->owner->getSiteConfig();
         return $config->ShareThisAllInOne;
     }
 
-    public function ShareAll()
+    /**
+     * @return boolean
+     */
+    public function getShareAll()
     {
         if ($this->IncludeShareAll()) {
             return ShareThisOptions::get_share_all();
         }
     }
 
     /**
-     * eturns array
+     * @return array
      */
     protected function makeShareIcons($bookmarks)
     {
-        $icons = array();
+        $icons = [];
         if ($bookmarks) {
-            $useFontAwesome = Config::inst()->get("ShareThisSTE", "use_font_awesome");
+            $useFontAwesome = Config::inst()->get(ShareThisSTE::class, "use_font_awesome");
             Requirements::themedCSS('SocialNetworking', "sharethis"); // ALSO  added in template
+
             if ($useFontAwesome) {
                 Requirements::css("//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css");
             }
-            Requirements::javascript(FRAMEWORK_DIR . '/thirdparty/jquery/jquery.js');
+
+            Requirements::javascript(SS_SHARETHIS_DIR . '/node_modules/jquery/dist/jquery.min.js');
```

Comment:
similar to previous comment about node_modules folder not available unless yarn install is ran.